layers/motorbike/motorbike_layer.vspec

```diff
@@ -0,0 +1,41 @@
+#
+# (C) 2022, Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+# This file contains adoptions to the main spec concering motorbikes.
+# It mainly addresses istantiation issues (e.g. wheels) and 
+# additional signals, which are motorbike specific (e.g. handlebars).
+
+
+# Branches used for the structure, not adopted in any way
+Vehicle:
+    type: branch
+
+Vehicle.Chassis:
+    type: branch
+
+Vehicle.Chassis.Axle:
+    type: branch
+
+# Changed instantiation
+Vehicle.Chassis.Axle.Wheel:
+    type: branch
+    instances: ["Center"]
+
+# Branches used for the structure, not adopted in any way
+Vehicle.Cabin:
```

Comment:
Maybe we should discuss how useful it is to keep the "cabin" branch for motorbikes. I believe only a very small subset of the signals (like seat heating) would be reusable. The term "cabin" is not intuitive for most bikes, but relevant for some like https://en.wikipedia.org/wiki/BMW_C1